internal/server/kong/ws/config/configuration.go

```diff
@@ -50,11 +52,33 @@ func (l *KongConfigurationLoader) Register(mutator Mutator) error {
 func (l *KongConfigurationLoader) Load(ctx context.Context, clusterID string) (Content, error) {
 	var configTable DataPlaneConfig = map[string]interface{}{}
 	for _, m := range l.mutators {
+		mutationStartTime := time.Now()
 		err := m.Mutate(ctx, MutatorOpts{ClusterID: clusterID},
 			configTable)
+		mutationDuration := time.Since(mutationStartTime).Milliseconds()
 		if err != nil {
+			metrics.Histogram("data_plane_config_mutation_time_individual",
+				float64(mutationDuration),
+				metrics.Tag{
+					Key:   "status",
+					Value: "fail",
+				},
+				metrics.Tag{
+					Key:   "mutator_name",
+					Value: m.Name(),
+				})
 			return Content{}, err
 		}
+		metrics.Histogram("data_plane_config_mutation_time_individual",
+			float64(mutationDuration),
+			metrics.Tag{
+				Key:   "status",
+				Value: "success",
+			},
+			metrics.Tag{
+				Key:   "mutator_name",
+				Value: m.Name(),
+			})
```

Comment:
This (and other instances of this) can be simplified: 😃 
```suggestion
		mutationStartTime := time.Now()
		err := m.Mutate(ctx, MutatorOpts{ClusterID: clusterID},
			configTable)
		metrics.Histogram(
			"data_plane_config_mutation_time_individual",
			float64(time.Since(mutationStartTime).Milliseconds()),
			metrics.Tag{Key: "status", Value: lo.Ternary(err == nil, "success", "fail")},
			metrics.Tag{Key: "mutator_name", Value: m.Name()},
		)
		if err != nil {
			return Content{}, err
		}
```